apps/meteor/app/api/server/v1/chat.js

```diff
@@ -524,6 +524,10 @@ API.v1.addRoute(
 	{
 		get() {
 			const { rid, type, text } = this.queryParams;
+			check(rid, String);
```

Comment:
Since now you have this, the check on L534 is no longer needed

Reply:
Technically you're right however I'd rather prefer using strings returned from the database rather than a user input while trying to generate a mongo query :grimacing:

Reply:
Oops! Sorry I misunderstood your comment earlier here :see_no_evil: I've removed that check now. Thanks!!